.travis.yml

```diff
@@ -7,7 +7,7 @@ services:
   - docker
 before_install:
   - docker pull trufflesuite/ganache-cli
-  - docker run -d -p 8545:8545 -p 3000:3000 -p 27017:27017 trufflesuite/ganache-cli

```

Comment:
ganache-cli docker image only exposes 8545, my guess is this was in there by mistake

Reply:
mikey is correct. it was wrong to begin with. ganache should use 8545, mongo container should use 27017, and the repo docker should use 3000.  mongo doesnt give an error in travisCI right now because the only tests are requesting the index, not what it returns.

Reply:
Ah, makes sense. Resolving this.